crates/zkwasm/src/foreign/context/runtime.rs

```diff
@@ -39,14 +41,20 @@ impl Context {
     }
 }
 
-impl ForeignContext for Context {}
+impl ForeignContext for Context {
+    fn get_statics(&self) -> Option<ForeignStatics> {
```

Comment:
give a default implement `get_statics` for `ForeignContext`

Reply:
implemented